### ISSUE TITLE
Bug fix for default phase angles of quasi stationary multi phase sources

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1099]
**Reported by christiankral on 14 May 2013 13:19 UTC**
The default phase angles of time transient multi phase voltage and currents sources 

 * Modelica.Electrical.MultiPhase.Sources.SineVoltage
 * Modelica.Electrical.MultiPhase.Sources.CosineVoltage
 * Modelica.Electrical.MultiPhase.Sources.SineCurrent
 * Modelica.Electrical.MultiPhase.Sources.CosineCurrent

are determined by  
```
phase=-Modelica.Electrical.MultiPhase.Functions.symmetricOrientation(m)
```

The default phase angles of the quasi stationary multi phase voltage and currents source

 * Modelica.Electrical.QuasiStationary.MultiPhase.Sources.VoltageSource
 * Modelica.Electrical.QuasiStationary.MultiPhase.Sources.CurrentSource

are currently determined by
```
phase={0 - (j - 1)*2*pi/m for j in 1:m}
```

The quasi stationary implementation is inconsistent. This is a bug and shall be fixed in MSL 3.2.1 and MSL 3.2.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1099

**Comment by modelica-trac-importer** [comment]
**Comment by christiankral on 14 May 2013 13:51 UTC**
Two comments:

 1) The problem in particular affects even number of phases only. For odd number of phases the original implementation and function `symmetricOrientation` lead to equal results.
 2) Bug fix will be applied to MSL 3.2.1 only, since multi phase systems with even number of phase became relevant with the multi phase machine models for m>3 -- which are introduced in MSL 3.2.1 (not 3.2)

**Comment by modelica-trac-importer** [comment]
**Comment by christiankral on 14 May 2013 13:54 UTC**
Fixed bug according to d41c9f46048c03ebb0eb9ba634fd634d9d49d140.
